### ISSUE TITLE
[FEATURE] Link Chatflows Together in a Super Chain or Master Flow

### ISSUE THREAD

**Issue by maximometascript** [FlowiseAI/Flowise#1269]
**Summary**
I would like to be able to link together the output of one chain directly into another one. 

**Additional context**
Currently, an agent has to handle multiple chains or tools in parallel, but I'd like to see more linear or sequential programming possible. This could be a sequencer forcing one chatflow to execute before another, a chatflow chain/agent which embeds one into another, or a new interface for linking up chatflows at a higher level.

**Examples**
For example, the user requests information on a student in a local database to obtain class records from the university.
Example 1: [SQL Chain]-->[Sequencer Prompt Template]-->[GET API Chain]
Example 2: [SQL Chain]-->[Chatflow Chain: GET API Chain]
Example 3: [SQL Chain Chatflow]-->[Template or Linear Programming Module]-->[GET API Chain Chatflow]

**Comment by HenryHengZJ** [comment]
yep, this is another feature we are working on! to allow more sequential flow. stay tuned!

**Comment by varunmehra5** [comment]
Was this pushed by any chance? Dify uses the same approach

**Comment by timur-nocodia** [comment]
This feature already pushed?

**Comment by fudu69** [comment]
Very interested with this feature? Is there any schedule?
